### PR TITLE
fix(VPCEP): vpcep service resource update document and optimize test case

### PR DESCRIPTION
**Pull request by deer-hang**
**What this PR does / why we need it**:
fix vpcep service resource and test and markdown.
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [X] Tests added/passed.
* [X] Documentation updated.
* [X] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/vpcep' TESTARGS='-run TestAccVPCEPService_Basic'
...
==> Checking that code complies with gofmt requirements... 
TF_ACC=1 go test ./huaweicloud/services/acceptance/vpcep -v -run TestAccVPCEPService_Basic -timeout 360m -parallel 4 
=== RUN   TestAccVPCEPService_Basic 
=== PAUSE TestAccVPCEPService_Basic
=== CONT  TestAccVPCEPService_Basic
--- PASS: TestAccVPCEPService_Basic (191.52s) 
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/vpcep     191.587s
```

### CODE REVIEW

File: docs/resources/vpcep_service.md [hunk 1]
```diff
@@ -33,25 +33,26 @@ The following arguments are supported:
...
 * `vpc_id` - (Required, String, ForceNew) Specifies the ID of the VPC to which the backend resource of the VPC endpoint
   service belongs. Changing this creates a new VPC endpoint service.
 
-* `server_type` - (Required, String, ForceNew) Specifies the backend resource type. The value can be **VM**, **VIP**
-  or **LB**. Changing this creates a new VPC endpoint service.
+* `server_type` - (Required, String, ForceNew) Specifies the backend resource type. The valid values are as follows:
+  + **VM**: Indicates the cloud server, which can be used as a server.
+  + **LB**: Indicates the shared load balancer, which is applicable to services with high access traffic and services
+    that require high reliability and disaster recovery.
+    Changing this creates a new VPC endpoint service.
```
Comment (ShiChangkuo) [inline docs/resources/vpcep_service.md]: it's better to add a new line

File: docs/resources/vpcep_service.md [hunk 2]
```diff
@@ -63,8 +64,8 @@ The following arguments are supported:
 
 The `port_mapping` block supports:
 
-* `protocol` - (Optional, String) Specifies the protocol used in port mappings. The value can be **TCP** or **UDP**. The
-  default value is **TCP**.
+* `protocol` - (Optional, String) Specifies the protocol used in port mappings. Only **TCP** supported.
+  The default value is **TCP**.
```
Comment (ShiChangkuo) [inline docs/resources/vpcep_service.md]:
```suggestion
* `protocol` - (Optional, String) Specifies the protocol used in port mappings. Only **TCP** is supported.
```
If only one value is allowed, we can default the default value

File: docs/resources/vpcep_service.md [hunk 3]
```diff
@@ -102,6 +103,6 @@ This resource provides the following timeouts configuration options:
```
Comment (houpeng80) [inline docs/resources/vpcep_service.md:103]: * `create` - Default is 10 minutes.
* `delete` - Default is 10 minutes.

File: docs/resources/vpcep_service.md [hunk 4]
```diff
@@ -63,13 +65,12 @@ The following arguments are supported:
 
 The `port_mapping` block supports:
 
-* `protocol` - (Optional, String) Specifies the protocol used in port mappings. The value can be **TCP** or **UDP**. The
-  default value is **TCP**.
+* `protocol` - (Optional, String) Specifies the protocol used in port mappings. Only **TCP** is supported.
```
Comment (ShiChangkuo) [inline docs/resources/vpcep_service.md]: it's better to put the optional parameter behind the required prams.

**Pull request merged**